---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true

---

{% include base_path %}

  You can also find my articles on my [Google Scholar](https://scholar.google.com/citations?user=NwdWAb4AAAAJ&hl=en) page
  
## 2024
### [Genomic epidemiology reveals geographical clustering of multidrug-resistant *Escherichia coli* ST131 associated with bacteraemia in Wales](https://doi.org/10.1038/s41467-024-45608-1)
&emsp; **White RT**, Bull MJ, Barker CR, Arnott JM, Wootton M, Jones LS, Howe RA, Morgan M, Ashcroft MM, Forde BM, Connor TR, Beatson SA. Genomic epidemiology reveals geographical clustering of multidrug-resistant *Escherichia coli* ST131 associated with bacteraemia in Wales. *Nature Communications*. 2024;15:1371 doi: [10.1038/s41467-024-45608-1](https://doi.org/10.1038/s41467-024-45608-1)

<span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-024-45608-1" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1038/s41467-024-45608-1" data-hide-no-mentions="true" class="altmetric-embed"> </span>


## 2023

### [Molecular characterisation of the Australian and New Zealand livestock *Chlamydia pecorum* strains confirms novel but clonal ST23 in association with ovine foetal loss](https://doi.org/10.1016/j.vetmic.2023.109774)

&emsp; Jelocnik M, **White RT**, Clune T, O’Connell J, Foxwell J, Hair S, Besier S, La T, Phillips N, Robbins A, Bogema S, Vaz PK, Legione AR, Jenkins C, Jacobson C. Molecular characterisation of the Australian and New Zealand livestock *Chlamydia pecorum* strains confirms novel but clonal ST23 in association with ovine foetal loss. *Veterinary Microbiology* 2023;109774 doi: [10.1016/j.vetmic.2023.109774](https://doi.org/10.1016/j.vetmic.2023.109774)

<span class="__dimensions_badge_embed__" data-doi="10.1016/j.vetmic.2023.109774" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.vetmic.2023.109774" data-hide-no-mentions="true" class="altmetric-embed"> </span>


### [Draft genomes of novel avian *Chlamydia abortus* strains from Australian Torresian crows (*Corvus orru*) shed light on possible reservoir hosts and evolutionary pathways](https://doi.org/10.1099/mgen.0.001134)

&emsp; Kasimov V, **White RT**, Jelocnik M. Draft genomes of novel avian *Chlamydia abortus* strains from Australian Torresian crows (*Corvus orru*) shed light on possible reservoir hosts and evolutionary pathways. *Microbial Genomics* 2023;9:001134 doi: [10.1099/mgen.0.001134](https://doi.org/10.1099/mgen.0.001134)

<span class="__dimensions_badge_embed__" data-doi="10.1099/mgen.0.001134" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1099/mgen.0.001134" data-hide-no-mentions="true" class="altmetric-embed"> </span>

### [The first genomic insight into *Chlamydia psittaci* sequence type (ST)24 from a healthy captive psittacine host in Australia demonstrates evolutionary proximity with strains from psittacine, human, and equine hosts](https://doi.org/10.1016/j.vetmic.2023.109704)

&emsp; **White RT**, Jelocnik M, Klukowski N, Haque MH, Sarker S. The first genomic insight into *Chlamydia psittaci* sequence type (ST)24 from a healthy captive psittacine host in Australia demonstrates evolutionary proximity with strains from psittacine, human, and equine hosts. *Veterinary Microbiology* 2023;109704 doi: [10.1016/j.vetmic.2023.109704](https://doi.org/10.1016/j.vetmic.2023.109704)

<span class="__dimensions_badge_embed__" data-doi="10.1016/j.vetmic.2023.109704" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1016/j.vetmic.2023.109704" data-hide-no-mentions="true" class="altmetric-embed"> </span>

### [Whole-genome sequencing of *Chlamydia psittaci* from Australasian avian hosts: A genomics approach to a pathogen that still ruffles feathers](https://doi.org/10.1099/mgen.0.001072)

&emsp; **White RT**, Kasimov V, Foxwell J, Jenkins C, Gedye K, Pannekoek Y, Jelocnik M. Whole-genome sequencing of *Chlamydia psittaci* from Australasian avian hosts: A genomics approach to a pathogen that still ruffles feathers. *Microbial Genomics* 2023;9:001072 doi: [10.1099/mgen.0.001072](https://doi.org/10.1099/mgen.0.001072)

<span class="__dimensions_badge_embed__" data-doi="10.1099/mgen.0.001072" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1099/mgen.0.001072" data-hide-no-mentions="true" class="altmetric-embed"> </span>

### [A discovery down under: decoding the draft genome sequence of *Pantoea stewartii* from Australia’s Critically Endangered western ground parrot/kyloring (*Pezoporus flaviventris*)](https://doi.org/10.1099/mgen.0.001101)

&emsp; **White RT**, Taylor W, Klukowski N, Vaughan-Higgins R, Williams E, Petrovski S, Rose JJA, Sarker S. A discovery down under: decoding the draft genome sequence of *Pantoea stewartii* from Australia’s Critically Endangered western ground parrot/kyloring (*Pezoporus flaviventris*). *Microbial Genomics* 2023;9:001101 doi: [10.1099/mgen.0.001101](https://doi.org/10.1099/mgen.0.001101)

<span class="__dimensions_badge_embed__" data-doi="10.1099/mgen.0.001101" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1099/mgen.0.001101" data-hide-no-mentions="true" class="altmetric-embed"> </span>

## 2022

### [One clone to rule them all: Culture-independent genomics of *Chlamydia psittaci* from equine and avian hosts in Australia](https://doi.org/10.1099/mgen.0.000888)

&emsp; **White RT**, Anstey SI, Kasimov V, Jenkins C, Devlin J, El-Hage C, Pannekoek Y, Legione AR, Jelocnik, M. One clone to rule them all: Culture-independent genomics of *Chlamydia psittaci* from equine and avian hosts in Australia. *Microbial Genomics* 2022;8:000888 doi: [10.1099/mgen.0.000888](https://doi.org/10.1099/mgen.0.000888)

<span class="__dimensions_badge_embed__" data-doi="10.1099/mgen.0.000888" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1099/mgen.0.000888" data-hide-no-mentions="true" class="altmetric-embed"> </span>

### [Phase variation in the glycosyltransferase genes of *Pasteurella multocida* associated with outbreaks of fowl cholera on free-range layer farms](https://doi.org/10.1099/mgen.0.000772)

&emsp; Omaleki L, Blackall PJ, Cuddihy T, **White RT**, Courtice JM, Turni C, Forde BM, Beatson SB. Phase variation in the glycosyltransferase genes of *Pasteurella multocida* associated with outbreaks of fowl cholera on free-range layer farms. *Microbial Genomics* 2022;8:000772 doi: [10.1099/mgen.0.000772](https://doi.org/10.1099/mgen.0.000772)

<span class="__dimensions_badge_embed__" data-doi="10.1099/mgen.0.000772" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1099/mgen.0.000772" data-hide-no-mentions="true" class="altmetric-embed"> </span>

## 2021

### [Completing the genome sequence of *Chlamydia pecorum* strains MC/MarsBar and DBDeUG: New insights into this enigmatic koala (*Phascolarctos cinereus*) pathogen](https://doi.org/10.3390/pathogens10121543)

&emsp; **White RT**, Legione AR, Taylor-Brown A, Fernandez CM, Higgins DP, Timms P, Jelocnik M. Completing the genome sequence of *Chlamydia pecorum* strains MC/MarsBar and DBDeUG: New insights into this enigmatic koala (*Phascolarctos cinereus*) pathogen. *Pathogens* 2021;10(12):1543 doi: [10.3390/pathogens10121543](https://doi.org/10.3390/pathogens10121543)

<span class="__dimensions_badge_embed__" data-doi="10.3390/pathogens10121543" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.3390/pathogens10121543" data-hide-no-mentions="true" class="altmetric-embed"> </span>
  
### [*Escherichia coli*: placing resistance to third-generation cephalosporins and fluoroquinolones in Australia and New Zealand into perspective](https://doi.org/10.1071/MA21031)

&emsp; **White RT**. *Escherichia coli*: placing resistance to third-generation cephalosporins and fluoroquinolones in Australia and New Zealand into perspective. *Microbiology Australia* 2021;42:104-110 doi: [10.1071/MA21031](https://doi.org/10.1071/MA21031)

<span class="__dimensions_badge_embed__" data-doi="10.1071/MA21031" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1071/MA21031" data-hide-no-mentions="true" class="altmetric-embed"> </span>


## 2020

### [Companion animals are spillover hosts of the multidrug-resistant human extraintestinal *Escherichia coli* pandemic clones ST131 and ST1193](https://doi.org/10.3389/fmicb.2020.01968)

&emsp; Kidsley AK, **White RT**, Beatson SA, Saputra S, Schembri MA, Gordon D, Johnson JR, O’Dea M, Mollinger JL, Abraham S, Trott DJ. Companion animals are spillover hosts of the multidrug-resistant human extraintestinal *Escherichia coli* pandemic clones ST131 and ST1193. *Frontiers in Microbiology* 2020;11:1968 doi: [10.3389/fmicb.2020.01968](https://doi.org/10.3389/fmicb.2020.01968) 

<span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2020.01968" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.3389/fmicb.2020.01968" data-hide-no-mentions="true" class="altmetric-embed"> </span>

## 2019

### [Genomic characterisation and context of the *bla*<sub>NDM-1</sub> carbapenemase in *Escherichia coli* ST101](https://doi.org/10.1101/860726)
  
&emsp; Ashcroft MM, Forde BM, Phan MD, Peters KM, Henderson A, Hancock SJ, Roberts LW, **White RT**, Chan KG, Chong TM, Yin WF, Paterson DL, Walsh TR, Schembri MA, Beatson SA. Genomic characterisation and context of the *bla*<sub>NDM-1</sub> carbapenemase in *Escherichia coli* ST101. bioRxiv:860726v2 [Preprint]. 2019. Available from: https://www.biorxiv.org/content/10.1101/860726v2 doi: [10.1101/860726](https://doi.org/10.1101/860726) 

<span class="__dimensions_badge_embed__" data-doi="10.1101/860726" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.1101/860726" data-hide-no-mentions="true" class="altmetric-embed"> </span>
  

Conference Posters
===================
  
## 2021

### [Genomic epidemiology reveals geographical clustering of multidrug-resistant *Escherichia coli* sequence type (ST)131 associated with bacteraemia in Wales, United Kingdom](http://dx.doi.org/10.13140/RG.2.2.12751.41122)
  
&emsp; **White RT**, Bull MJ, Barker CR, Arnott JM, Wootton M, Jones LS, Howe RA, Morgan M, Ashcroft MM, Forde BM, Connor TR, Beatson SA. Genomic epidemiology reveals geographical clustering of multidrug-resistant *Escherichia coli* sequence type (ST)131 associated with bacteraemia in Wales, United Kingdom. Poster presented at: Australian Society for Microbiology (ASM) 2021. Melbourne, Australia; 2021 doi: [10.13140/RG.2.2.12751.41122](http://dx.doi.org/10.13140/RG.2.2.12751.41122)

<span class="__dimensions_badge_embed__" data-doi="10.13140/RG.2.2.12751.41122" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.13140/RG.2.2.12751.41122" data-hide-no-mentions="true" class="altmetric-embed"> </span>
  
## 2018

### [Capsule-switching is associated with the rapid global expansion of the recently emerged fluoroquinolone-resistant *Escherichia coli* sequence type 1193 clone](http://dx.doi.org/10.13140/RG.2.2.32515.43049)
  
&emsp; **White RT**, Forde BM, Ashcroft MM, Roberts LW, Trott DJ, Gibson J, Mollinger JL, Rogers BA, Kidsley AK, Bell J, Turnidge J, Schembri MA, Beatson SA. Capsule-switching is associated with the rapid global expansion of the recently emerged fluoroquinolone-resistant *Escherichia coli* sequence type 1193 clone. Poster presented at: Australian Society for Microbiology (ASM) 2018. Brisbane, Australia; 2018 doi: [10.13140/RG.2.2.32515.43049](http://dx.doi.org/10.13140/RG.2.2.32515.43049)

<span class="__dimensions_badge_embed__" data-doi="10.13140/RG.2.2.32515.43049" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.13140/RG.2.2.32515.43049" data-hide-no-mentions="true" class="altmetric-embed"> </span>
  
## 2017

### [Genomic characterisation of *Escherichia coli* ST1193: an emerging fluoroquinolone-resistant variant of an O75-associated extra-intestinal pathogenic clonal group](http://dx.doi.org/10.13140/RG.2.2.20771.37928)
  
&emsp; **White RT**, Forde BM, Roberts LW, Trott DJ, Gibson J, Mollinger JL, Schembri MA, Beatson SA. Genomic characterisation of *Escherichia coli* ST1193: an emerging fluoroquinolone-resistant variant of an O75-associated extra-intestinal pathogenic clonal group. Poster presented at: BacPath14. Hahndorf, Australia; 2017 doi: [10.13140/RG.2.2.20771.37928](http://dx.doi.org/10.13140/RG.2.2.20771.37928)
  
<span class="__dimensions_badge_embed__" data-doi="10.13140/RG.2.2.20771.37928" data-style="small_circle" data-hide-zero-citations="true"> <span data-badge-popover="right" data-badge-type="donut" data-doi="10.13140/RG.2.2.20771.37928" data-hide-no-mentions="true" class="altmetric-embed"> </span>
  
  
  
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
